Library/src/Systems/ECommerce/ECommerceSystem.cpp

```diff
@@ -126,6 +126,28 @@ void ECommerceSystem::GetCart(const common::String& SpaceId, const common::Strin
 	static_cast<chs::ShopifyApi*>(ShopifyAPI)->apiV1SpacesSpaceIdVendorsShopifyCartsCartIdGet(SpaceId, CartId, ResponseHandler);
 }
 
+void ECommerceSystem::GetShopifyStores(const common::String& UserId, const csp::common::Optional<bool>& IsActive, GetShopifyStoresResultCallback Callback)
+{
+	csp::services::ResponseHandlerPtr ResponseHandler
+		= ShopifyAPI->CreateHandler<GetShopifyStoresResultCallback, GetShopifyStoresResult, void, csp::services::DtoArray<chs::ShopifyStorefrontDto>>(
+			Callback,
+			nullptr,
+			csp::web::EResponseCodes::ResponseCreated);
+
+	std::optional<bool> ActiveParam;
+
+    if (IsActive.HasValue())
```

Comment:
Not a reason to block this PR, but just as a note for the future - the logic here is equivalent to just passing `IsActive` straight through to the endpoint call, no need for a local optional variable here.